src/components/routes/Booking/BookingPayment/BookingPaymentButton.tsx

```diff
@@ -21,14 +23,16 @@ interface State {
   isSubmitting: boolean;
 }
 
+const FROM_BEE_MESSAGE = 'You will be prompted to Confirm two Ethereum transactions to complete this payment.';
```

Comment:
Would this alert the user more?
``You will be prompted to Confirm two Ethereum transactions`` => 
``You will be prompted to confirm TWO Ethereum transactions`` ?